stdlib/logging/__init__.pyi

```diff
@@ -740,18 +775,23 @@ class RootLogger(Logger):
 
 root: RootLogger
 
-class PercentStyle(object):
+class PercentStyle(object):  # undocumented
     default_format: str
     asctime_format: str
     asctime_search: str
+    validation_pattern: Any
```

Comment:
`validation_pattern` seems to be new in Python 3.8. The following should work here (if `re` is imported):

```suggestion
    if sys.version_info >= (3, 8):
        validation_pattern: re.Pattern[str]
```